Update to correct Iris file

ranieuwe opened the pull request:
The generated value is iris-1.dprep, not iris.dprep, because the file we just generated is iris-1.dprep.

azurecla commented:
Hi __@brohrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Do not approve this PR. Updates pending.

rmca14 commented:
Updates made in PR #10959.

ranieuwe committed:
Update to correct Iris file

The generated value is iris-1.dprep, not iris.dprep, because the file we just generated is iris-1.dprep.
